traj/scripts/traj_segment_example.py

```diff
@@ -0,0 +1,89 @@
+#!/usr/bin/env python
+"""
+Simple example that fits general trajectory segment, given the start and
+end positions/velocities. Start and end acceleration/jerk are assumed to be zero.
+"""
+from matplotlib import pyplot as plt
+import traj
+
+
+def test_seven_segment(p_start, p_end, v_start, v_end, p_max, v_max, a_max, j_max):
+    position, velocity, acceleration, jerk = traj.fit_traj_segment(p_start, p_end, v_start, v_end, p_max, v_max, a_max, j_max)
+    traj.plot.plot_trajectory(plt, position, velocity, acceleration, jerk, n_points=100, v_max=v_max, a_max=a_max, j_max=j_max)
+    plt.show()
+    
+p_max=30
+v_max=3.0
+a_max=4.0
+j_max=10.0   
+
+############### CASE A: v_start = v_end = 0    
+#case A1:  no limit is reached     
```

Comment:
This is an awesome set of tests. Would you mind putting this in the `traj/test` directory and making it a nosetest? Then we can run it automatically as part of the test suite.

Reply:
I already added 2 test files with all of these cases in traj/test directory: [test_traj_segment.py](https://github.com/mahmoud-a-ali/trajectory_smoothing/blob/17cfb09e75688e05bf462de2a7d69d229ee429cc/traj/test/test_traj_segment.py#L1), and [test_traj_segment_middle_points.py ](https://github.com/mahmoud-a-ali/trajectory_smoothing/blob/17cfb09e75688e05bf462de2a7d69d229ee429cc/traj/test/test_traj_segment_middle_points.py#L1)